test/cli/other_test.py

```diff
@@ -2784,4 +2784,54 @@ def test_addon_suppr_cli_file_line(tmp_path):
 
 def test_addon_suppr_cli_absfile_line(tmp_path):
     test_file = tmp_path / 'test.c'
-    __test_addon_suppr(tmp_path, ['--suppress=misra-c2012-2.3:{}:3'.format(test_file)])
\ No newline at end of file
+    __test_addon_suppr(tmp_path, ['--suppress=misra-c2012-2.3:{}:3'.format(test_file)])
```

Comment:
Shouldn't this be in `whole-program_test.py`?

Reply:
Logically it does. But (at least in my book) the grouping of those tests is around the `whole-program` folder in the repo (similar to `proj2`, `helloword`, etc.).

But `other_test.py` should probably to be split up at some point as it just a dumping ground for random tests. But that is not something I am willing to spend any time on soon with all the pending stuff.

Reply:
Weeell... You could add a file in `whole-program` instead of creating it on-the-fly.

Reply:
That would start another dumping ground. IMO we should only create physical files if necessary (headers, folders, ...) or if they have a certain complexity/size.

Reply:
Fair enough. I'd still prefer the new test in `whole-program_test.py` though.

Reply:
> That would start another dumping ground.

One one side I also spontanously think all tests related to whole program analysis should be in the same test file. But I can understand your logic too.

> IMO we should only create physical files if necessary

Certainly.

Reply:
It's not something you should do here in this PR.. it's just an observation.

The other_test.py has many tests related to suppressions and therefore I feel it would make sense to create a testfile for all suppression tests.

Reply:
@chrchr-github Can we merge this and shuffle around the tests later?